h/models/token.py

```diff
@@ -24,6 +24,9 @@ class Token(Base, mixins.Timestamps):
     #: to, for example, one of the short-lived JWTs that the client uses).
     prefix = u'6879-'
 
+    #: A prefix that identifies a token as a refresh token.
+    refresh_token_prefix = u'7980-'
```

Comment:
> If refresh tokens have a prefix then we can easily identify mistakes such as using the refresh token in the place where the access token is supposed to be, that sort of thing

We can easily do this by just the token having a prefix and not the refresh token. It would also eliminate complexity from the model.